tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2015",
```

Comment:
This is probably my being new to Typescript, but do we not need to target es5? Or are we assuming users of our library will transpile our es6/es2015 as part of their build?

And same question for the es2017 in [packages/cli/tsconfig.json](https://github.com/blitz-js/blitz/pull/8/files#diff-fea82da13cd5f2a75bd510aaab6f18ad)

Reply:
Yes, we need to leave this as `es5` like @medelman17 set up. If the CLI has different needs, then overside the top level tsconfig in the CLI tsconfig.

Reply:
I reverted the change. I'm just curious, why do we need to compile down to es5? The latest node LTS supports es2019

Reply:
@lorenzorapetti `es5` is for the other packages, not the cli. The cli can target something newer.